components/ui/animated-background.tsx

```
@@ -0,0 +1,90 @@
+        if (onValueChange) {
+            onValueChange(id);
+        }
+    };
+
+    useEffect(() => {
+        if (defaultValue !== undefined) {
+            setActiveId(defaultValue);
+        }
+    }, [defaultValue]);
+
+    return Children.map(children, (child: any, index) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)